tools/report-converter/codechecker_report_converter/cli.py

```diff
@@ -14,18 +14,25 @@
 import os
 import shutil
 
+# If we run this script in an environment where 'codechecker_report_converter'
+# module is not available we should add the grandparent directory of this file
+# to the system path.
```

Comment:
It looks good to me, but please add a TODO here that this section will not be needed when CodeChecker will be delivered as a python package and will be set up as a venv.